bz-manage.sh

```diff
@@ -381,4 +382,137 @@ get_bz_id_summary() {
   done
 }
 
+update_doc_text() {
+  local queryurl jq bz bz_assignee doc_type doc_text gh_pr gh_pr_info gh_pr_title gh_pr_desc gh_pr_type update_bz
+  query_file=.query_file.json
+
+  # If a BZ has required_doc_text unset and GH PR set:
+  # Automatically add doc text from PR description
+  queryurl="${BASE_URL}"
+  queryurl+="&f1=flagtypes.name&f2=external_bugzilla.url&f3=ext_bz_bug_map.ext_bz_bug_id"
+  # substring doesn't work with ext_bz_bug_map.ext_bz_bug_id, using anywordssubstr
+  queryurl+="&o1=notsubstring&o2=substring&o3=anywordssubstr"
+  queryurl+="&v1=requires_doc_text&v2=github&v3=pull"
+  jq='.bugs[] | (.id | tostring) + '
+  jq+='" https://github.com/" + (.external_bugs[] | select(.type.type == "GitHub") | .ext_bz_bug_id | tostring)'
+  get_bzs "${queryurl}" "$jq" -r | while read -r bz gh_pr; do
+    echo "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
+    gh_pr_info=$(gh pr view "$gh_pr" --json title,body --jq .title,.body)
+    gh_pr_title=$(echo "$gh_pr_info" | sed -n 1p)
+    gh_pr_desc=$(echo "$gh_pr_info" | sed 1d)
+    if [[ "$gh_pr_title" =~ ^feat.* ]]; then
+      gh_pr_type="Enhancement"
+    elif [[ "$gh_pr_title" =~ ^fix.* ]]; then
+      gh_pr_type="Bug Fix"
+    fi
+    comment="Updating Doc Type and Text with info from the attached PR $gh_pr"
+    echo "BZ#$bz: $comment"
+    echo bugzilla modify --private \
+      --field cf_doc_type="$gh_pr_type" \
+      --field cf_release_notes="$gh_pr_desc" \
```

Comment:
The $gh_pr_desc is always multiple lines of text. How do I make it work here? I'd think I can use env variable, but bugzilla cli doesn't allow for it.

Reply:
You should be able to pass a multiline string in a double quoted variable